db/migrate/20170504145330_sync_whitehall_public_updated_at.rb

```diff
@@ -0,0 +1,29 @@
+class SyncWhitehallPublicUpdatedAt < ActiveRecord::Migration[5.0]
+  def change
+    schema_names = %w(gone redirect special_route speech statistical_data_set take_part topical_event_about_page)
+
+    schema_names.each do |schema_name|
+      scope = Edition.where(publishing_app: "whitehall")
+                     .where(schema_name: schema_name)
+                     .where("first_published_at AT TIME ZONE 'Europe/London' > public_updated_at")
+
+      # Sync all the timestamps
+      scope.update_all("public_updated_at = first_published_at")
+      puts "#{scope.count} records updated for schema '#{schema_name}'"
+      puts "(#{scope.pluck(:id).join(",")})"
+
+      # Narrow the scope to find current content
+      current_editions_scope = scope.where(state: %w(draft published))
+      current_edition_content_ids = current_editions_scope.map(&:content_id)
+
+      if current_editions_scope.any?
+        # Represent current editions downstream
+        Commands::V2::RepresentDownstream.new.call(current_edition_content_ids, with_drafts: true)
```

Comment:
Because you are doing this, you'll want to include the following line at the top of the class:

```ruby
disable_ddl_transaction!
```